### PR TITLE
Changes to help with validation

### PR DESCRIPTION
**Pull request by wajiha-oicr**
**Description of changes**

1. Include the name of the field as a variable that the validation scripts can use.
2. Make invalid_by_script show what the error message from the script returned. I think that may fix [this bug](https://github.com/icgc-argo/argo-dictionary/issues/35).

**Type of Change**

- [ ] Bug
- [x] Refactor
- [ ] New Feature

**Checklist before requesting review:**

- [x] Check branch (code change PRs go to `develop` not master)
- [x] Manual testing
- [ ] Regression tests completed and passing (double check number of tests).
- [x] Spelling has been checked.
- [ ] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

File: src/lectern-client/schema-error-messages.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ const ERROR_MESSAGES: { [key: string]: (errorData: any) => string } = {
   INVALID_FIELD_VALUE_TYPE: () => INVALID_VALUE_ERROR_MESSAGE,
   INVALID_BY_REGEX: () => INVALID_VALUE_ERROR_MESSAGE,
   INVALID_BY_RANGE: () => 'Value is out of permissible range',
-  INVALID_BY_SCRIPT: () => INVALID_VALUE_ERROR_MESSAGE,
+  INVALID_BY_SCRIPT: (error) => error.info.message,
```
Comment (blabadi) [inline src/lectern-client/schema-error-messages.ts]: Cool, this is the way it should be ! I didn't know this was changed

**Comment by blabadi** [comment]
did you want to add the error log in the catch 
```
} catch (err) {
      console.error(`failed running validation script ${field.name} for record: ${record}`);
}
```

**Comment by wajiha-oicr** [comment]
> did you want to add the error log in the catch
> 
> ```
> } catch (err) {
>       console.error(`failed running validation script ${field.name} for record: ${record}`);
> }
> ```

Oh right. Added it in new [commit](https://github.com/icgc-argo/argo-clinical/pull/355/commits/0e341d2f2943d63c69c1ab5c85de9fa64f145b73).

**Comment by blabadi** [comment]
do you know how to restart the tests in jenkins ?

**Comment by wajiha-oicr** [comment]
> do you know how to restart the tests in jenkins ?

No I do not. I actually don't know if I even have access to jenkins. Clicking the links redirects me to a login page.

**Pull request merged**